scripts/create-kind-cluster-with-SSL-passthrough.sh

```diff
@@ -117,6 +117,9 @@ else
   echo "Skipping... \"${name}\" kind cluster already exists."
 fi
 
+echo "Waiting for cluster nodes to be ready..."
+kubectl --context "kind-${name}" wait --for=condition=ready nodes --all --timeout=300s
```

Comment:
I do not understand why this wait is needed. The following commands do not depend directly on Node readiness, at most they depend indirectly on Node readiness --- so the existing waits on what really matters should cover Node readiness.